curvefs/src/client/rpcclient/cli2_cient.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/*
+ * Project: curve
+ * Created Date: Mon Sept 4 2021
+ * Author: lixiaocui
+ */
+
+#include "curvefs/src/client/rpcclient/cli2_client.h"
+
+namespace curvefs {
+namespace client {
+namespace rpcclient {
+
+void Cli2Closure::Run() {
+    bool success = false;
+    if (cntl.Failed()) {
+        LOG(WARNING) << "GetLeader failed from " << cntl.remote_side()
```

Comment:
Cli2Closure seems to be a common closure, it is inappropriate to use  "GetLeader failed" in log